ci/Vagrantfile

```diff
@@ -0,0 +1,37 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+require 'etc'
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "base"
+
+  if Vagrant.has_plugin?('vagrant-env')
+    config.env.enable
+  end
+
+  machine_name = ENV['MACHINE_NAME'] || ("vmck-vagrant-" + `hostname`.strip)
+  custom_sh = ENV['PROVISION_SH']
+
+  config.vm.define machine_name
+  config.nfs.functional = false
+
+  config.vm.synced_folder "../.", "/opt/vmck", type: "rsync",
+      rsync__exclude: ["../.git"],
```

Comment:
Are you sure this works? Double underscore? And I'm pretty sure "rsync exclude" matches on folder names, so it won't be happy if you give it a path. But don't take my word for it, better to test it yourself.